roles/basic/files/20-update

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/python
```

Comment:
Diese Datei kopiert das Verhalten von unattended-upgrades, das bereits installiert wird.

Reply:
Was machen wir hiermit? Ich habe gesehen, dass das in diesem Playbook noch nicht drin ist, aber dafür gibt es eine Rolle, die ich dann in einem PR hinzufügen würde.